notebooks/end-to-end-structured/solutions/5a_train_keras_vertex_babyweight.ipynb

```diff
@@ -0,0 +1,926 @@
+{
```

Comment:
<ul><li>Please add this dependency in the <code>requirements.txt</code> at the repo top-level </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>+1</p>

Reply:
<p>I moved it to requirements.txt. Thanks!</p>